src/controllers/product/index.js

```diff
@@ -0,0 +1,3 @@
+import { product } from '~/src/models'
+
+export const products = () => product.findAll()
```

Comment:
Functions should be named after verbs.

Reply:
can be a verb like: getProducts?

Reply:
That's right.

Reply:
Fixed!, also I've created an Internal error type to handle errors that client should not know. Please check it out